### DISCUSSION TITLE
ISSUE RUNNING CODE

### DISCUSSION THREAD

**Discussion by AyJobee1** [giswqs/geog-312#98] [Q&A] [answered]
While following the Week 6 lecture, I ran into this error: the CRS from the meta is different from the one displayed in your video.
Which results to subsequent errors when trying to parse the src.crs to gdf. Please, how can I resolve this?
![Screenshot 2024-10-05 215045](https://github.com/user-attachments/assets/67028c50-49c4-4cc8-a838-befddec22650)
 Subsequent error trying to parse the crs
![Screenshot 2024-10-05 214120](https://github.com/user-attachments/assets/c432c91c-ed92-40c7-87ac-dd1caca2f446)

Thank you

**Comment by giswqs** [comment, marked as answer]
It works fine on my end. Try Google Colab to see if you encounter the same issue.
